schematron/lib/tests/data/ex01.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0"?>
```

Comment:
Do you know if this is the same file as any of the other example files we have?  Maybe we can put something more descriptive as the file name, or just use a different file?  If it is different, then that is fine.  Just trying not to have duplicate files with different names.

Reply:
It's a modified version of spec/files/good/L100_Copy.xml. I will remove those original files once the tests are moved into python